test/docfx.Test/lib/MonikerRangeParserTest.cs

```diff
@@ -117,14 +117,14 @@ public void TestEvaluateMonikerRange(string rangeString, string expectedMonikers
         }
 
         [Theory]
-        [InlineData("netcore-xp", "Invalid moniker range 'netcore-xp': Moniker 'netcore-xp' is not defined.")]
-        [InlineData("netcore-1.0 < || netcore-2.0", "Expect a moniker string, but got ' || netcore-2.0'.")]
-        [InlineData(">netcore&-1.0", "Parse ends before reaching end of string, unrecognized string: '&-1.0'.")]
-        [InlineData(">=>netcore&-1.0", "Expect a moniker string, but got '>netcore&-1.0'.")]
-        [InlineData(">netcore<-1.0", "Invalid moniker range '>netcore<-1.0': Moniker 'netcore' is not defined.")]
-        [InlineData(">netcore<-1.0 ||| >netcore-2.0", "Expect a comparator set, but got '| >netcore-2.0'.")]
-        [InlineData(">netcore<-1.0 || ||", "Expect a comparator set, but got ' ||'.")]
-        [InlineData(">netcore<-1.0 || <", "Expect a moniker string, but got ''.")]
+        [InlineData("netcore-xp", "Invalid moniker range: 'netcore-xp'. Moniker 'netcore-xp' is not defined.")]
```

Comment:
```suggestion
        [InlineData("netcore-xp", "Invalid moniker range: Moniker 'netcore-xp' is not defined.")]
```

Reply:
```
        [InlineData("netcore-xp", "Invalid moniker range: 'netcore-xp'. Moniker 'netcore-xp' is not defined.")]
        [InlineData(">netcore<-1.0", "Invalid moniker range: '>netcore<-1.0'. Moniker 'netcore' is not defined.")]
```
If I changed first line, then second line will be 
```
[InlineData(">netcore<-1.0", "Invalid moniker range: Moniker 'netcore' is not defined.")]
```
They uses same string template. Is this error message accepted?

Reply:
How about change the MonikerRangeInvalid message template to `Invalid moniker range '{monikerRange}': {message}`?

Reply:
some callers have no `monikerRange` parameter. If so, 
```
[InlineData(">netcore<-1.0 ||| >netcore-2.0", "Invalid moniker range: Expect a comparator set, but got '| >netcore-2.0'.")]
```
will become into:
```
[InlineData(">netcore<-1.0 ||| >netcore-2.0", "Invalid moniker range '| >netcore-2.0': Expect a comparator set, but got '| >netcore-2.0'.")]
```
Does this make sense?